src/__tests__/render.test.tsx

```diff
@@ -191,11 +211,121 @@ test('debug changing component', () => {
   // eslint-disable-next-line no-console
   const mockCalls = (console.log as any as ConsoleLogMock).mock.calls;
 
-  expect(stripAnsi(mockCalls[4][0])).toMatchSnapshot(
+  expect(stripAnsi(mockCalls[0][0])).toMatchSnapshot(
     'bananaFresh button message should now be "fresh"'
   );
 });
 
+test('debug with only children prop', () => {
+  jest.spyOn(console, 'log').mockImplementation((x) => x);
+
+  const { debug, getByText } = render(<Banana />);
+
+  debug({ mapProps: ({ children }) => ({ children }) });
```

Comment:
This test seems confusing, as `children` in not included in the props. Pls adjust or remove the test.

Reply:
Oh you're right, I had written them before I realized the children prop wasn't filtered and it does not really make sense anymore. I deleted one test and modified the other one